examples/lifecycleDemo.html

```diff
@@ -115,9 +115,52 @@ <h1>Lifecycle Events Log:</h1>
             var lifecycleStatusMessageTitle = 'Lifecycle Demo App';
             var lifecycleStatusMessageId = 'lifecycleDemo-statusMsg';
 
+            function extractParams(paramStr) {
+                let result = {};
+
+                if (paramStr) {
+                    let params = paramStr.split('&');
+                    params.forEach(function(currParam) {
+                        if (currParam) {
+                            let paramTokens = currParam.split('=');
+                            let paramName = paramTokens[0];
+                            let paramValue = paramTokens[1];
+                            if (paramName) {
+                                paramName = decodeURIComponent(paramName);
+                                paramValue = paramValue ? decodeURIComponent(paramValue) : null;
+
+                                if (!result.hasOwnProperty(paramName)) {
+                                    result[paramName] = paramValue;
+                                } else if (Array.isArray(result[paramName])) {
+                                    result[paramName].push(paramValue);
+                                } else {
+                                    result[paramName] = [result[paramName], paramValue];
+                                }
+                            }
+                        }
+                    });
+                }
+
+                return result;
+            }
+
+            function getRequireManualToastDismissal() {
+                let result = null;
+                if (window.location.search) {
+                    result = extractParams(window.location.search.substring(1)).requireManualToastDismissal;
+                    if (result === 'true') {
+                        return true;
+                    }
+                }
+                return false;
+            }
+
             jQuery(function () {
                 let myClientApp = null;
 
+                var requireManualToastDismissal = getRequireManualToastDismissal() === 'true' ? true : false;
```

Comment:
This line won't work now.  You should just use the return value of the function now that it directly returns the boolean.